src/models/user.js

```diff
@@ -4,8 +4,8 @@ module.exports = User;
 
 function User(attr) {
 	// TODO: Remove this
-	attr.name = attr.nick;
-	attr.mode = attr.modes[0] || "";
+	attr.name = attr.name || attr.nick;
+	attr.mode = attr.mode || (attr.modes && attr.modes[0]) || "";
```

Comment:
I added changes here simply because of the TODO above. If the model changes because irc-framework makes us too, I think we should go all in: `name` could be reworded `nick` everywhere and the non-array `mode` could disappear.
In the meantime, it was an easy way for me to integrate the tests, until this TODO gets addressed.
@xPaw, what's your take about changing the model regarding `name`/`nick`/`mode`/`modes`?